pylearn2/train_extensions/tests/test_monitor_based_save_best.py

```diff
@@ -44,14 +45,14 @@ def test_tagging():
 
         # Test that the default key gets created.
         def_model = MockModel()
-        def_model.monitor = MockMonitor()
+        def_model.monitor = Monitor(def_model)
```

Comment:
ok, then perhaps you can remove MockMonitor all together if it's not being used.